bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/i18n/nikohomecontrol_fr.properties

```diff
@@ -24,8 +24,8 @@ bridge2ConfigPasswordLabel = Jeton API
 bridge2ConfigPasswordDescription = Le jeton pour l'API hobby Niko Home Control II, ne doit pas être vide. Ce jeton devra être renouvelé après l'expiration (1 an après la création)
 
 bridgeConfigRefreshLabel = Fréquence de rafraîchissement
-bridgeConfigRefreshDescription = Intervalle de rafraîchissement pour la connexion avec l'interface IP de Niko Home Control (minutes), par défaut 300. Si défini à 0 ou laissé vide, aucun rafraîchissement ne sera programmé
-bridge2ConfigRefreshDescription = Intervalle de rafraîchissement pour la connexion avec l'unité de contrôle connectée ou le hub intelligent sans fil (minutes), par défaut 300. Si défini à 0 ou laissé vide, aucun rafraîchissement ne sera programmé
+bridgeConfigRefreshDescription = Intervalle de rafraîchissement pour la connexion avec l'interface IP de Niko Home Control (minutes), par défaut 300. Si défini à 0, aucun rafraîchissement ne sera programmé
```

Comment:
Not to be corrected in that PR but rather later in Crowdin.

Reply:
OK